pxtblocks/codecardRenderer.ts

```diff
@@ -100,7 +100,13 @@ export function renderCodeCard(card: pxt.CodeCard, options: CodeCardRenderOption
         }
         if (card.description) {
             const descr = div(ct, 'ui description');
-            const shortenedDescription = card.description.split('.')[0] + '.';
+            const regex = /((?:\.{1,3})|[\!\?…])/;
```

Comment:
This is a nice improvement, also somewhat English-centric. Are there additional characters we should consider here to support all languages?

Reply:
(totally fine to defer additional character support if it isn't straight forward)

Reply:
I'll look at the languages we support in the editor and see what else needs to be added.

Reply:
Yea I wasn't able to find a collection of end-of-sentence punctuation for all languages. This would require going through each language one by one. I'm going to defer this work for now.

Reply:
if we are concerned about this, we could always only enable this behavior for languages we know it works for and then just use css text-overflow for the rest

Reply:
@riknoll How would I do that?